src/tracing/instrumentation/httpServer.js

```diff
@@ -65,10 +66,12 @@ function requestListener(req, res) {
   }
 
   res.on('finish', function() {
+    var urlParts = url.parse(req.url);
```

Comment:
I noticed before that this is a really expensive operation and removed it in some location within this module (also within other Instana components). Alternatively, could we look for the first `?` and grab everything after that till the end of the string?

Reply:
I was wondering why you didn't use it...